eng/helix/content/runtests.sh

```diff
@@ -92,4 +92,8 @@ echo "Running tests: $DOTNET_ROOT/dotnet run --project RunTests/RunTests.csproj
 $DOTNET_ROOT/dotnet run --project RunTests/RunTests.csproj -- --target $1 --sdk $2 --runtime $3 --queue $4 --arch $5 --quarantined $6 --ef $7 --aspnetruntime $8 --aspnetref $9 --helixTimeout ${10}
 exit_code=$?
 echo "Finished tests...exit_code=$exit_code"
+
+# dotnet-install.sh leaves the temporary SDK archive on the helix machine which slowly fills the disk, we'll be nice and clean it until the script fixes the issue
+rm -r -f ${TMPDIR:-/tmp}/dotnet.*
```

Comment:
@MattGal I'm surprised `$TMPDIR` et cetera aren't redirected in the Helix runs. Isn't this an issue that should be handled in the Arcade repo?